c/blake3_llfio.cpp

```diff
@@ -0,0 +1,103 @@
+#include <variant>
+
+#include <llfio.hpp>
+
+#include "blake3.h"
+#include "blake3_impl.h"
+
+namespace llfio = LLFIO_V2_NAMESPACE;
+
+template <class... Ts> struct overloads : Ts... {
+  using Ts::operator()...;
+};
+// NOTE: deduction guide only needed for Apple Clang now.
+template <class... Ts> overloads(Ts...) -> overloads<Ts...>;
+
+INLINE auto copy_wide(blake3_hasher *self, llfio::path_view path,
+                      llfio::file_handle &file) noexcept
+    -> llfio::result<std::uint64_t> {
+  std::array<std::byte, 65535> buffer{};
```

Comment:
It's actually important for performance that the buffer size is a power of 2. Can we use 65536 for the size here, as in the Rust code?